src/calculateRentalCost.js

```diff
@@ -5,6 +5,19 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
+  const x = price * days;
+  if (days >= 7) {
```

Comment:
2. [checklist] Don't use literals in expressions (HARDCODE). Instead use constants explaining this values

Reply:
good point 🔥 
do it!
use [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) for it